tests/alembic_migrations/versions/v_7b139906ac46_add_not_null_constraint_on_meta_vars/test_smoke.py

```diff
@@ -0,0 +1,58 @@
+"""Smoke tests:
+- Upgrade adds not null constraint
+- Downgrade drops not null constraint
+"""
+
+# -*- coding: utf-8 -*-
+import logging
+import pytest
+from alembic import command
+from sqlalchemy import inspect
+
+logger = logging.getLogger("tests")
+
+@pytest.mark.parametrize(
+        "prepared_schema_from_migrations_left", ("3d50ec832e47",),indirect=True
+)
+@pytest.mark.usefixtures("new_db_left")
+def test_upgrade(
+    prepared_schema_from_migrations_left, alembic_config_left, schema_name
+):
+    """test migration from 3d50ec832e47 to 7b139906ac46. """
+    
+    # Set up database to version 7b139906ac46
+    engine, script = prepared_schema_from_migrations_left
+
+    engine.execute(
+        f"INSERT INTO {schema_name}.meta_vars(vars_id)"
+        f"VALUES (1)"
+    )
+
+    command.upgrade(alembic_config_left, "+1")
+
```

Comment:
After the migration check that the nulls in the row(s) you inserted have been set to the expected non-null values (or just not null).